LICENSE

```diff
@@ -1,6 +1,6 @@
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```

Comment:
I dunno if it matters, but https://www.apache.org/licenses/LICENSE-2.0.txt (i.e. the license text itself, which this file is a copy of) still says `http://` rather than `https://`